src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/HttpRequestMethods.java

```diff
@@ -0,0 +1,24 @@
+package com.amazon.opendistroforelasticsearch.security.securityconf.impl;
+
+public enum HttpRequestMethods {
```

Comment:
We cannot reuse elasticsearch org.elasticsearch.rest.RestRequest.Method enum ?

Reply:
I initially tried doing this. This is not possible. You need an enum with String values, since the yml files contents are parsed as strings.